README.md

```diff
@@ -77,20 +80,20 @@ This experience can post to [Twitter](https://twitter.com/GCPEmotobooth/status/7
 
 ## JSON & Grid Options
 
-There are several querystrings that can be added to the main page's url to affect its appearance and behavior. Default behavior, i.e., what happens without a querystring, is what we would want in a production environment.
+There are several querystrings that can be added to the grid page's url to affect its appearance and behavior. Default behavior (what happens without a querystring) is what we would want in a production environment.
 
-- `showgrid`: **the exception; we will want to use this on one of the production screens** shows the grid / historical view rather than the json / latest view.
-- `zoom`: resize UI so panels can fit, based on width, in your window.
-- `controls`: in json view, adds toolbar with shortcuts to taking certain photos to bottom of screen
-- `dontprint`: tells the application not to render final photostrip to the `site/out-print` folder
-- `prepopulate`: in grid view, uses pregenerated images to fill out grid if we don't actually have enough historical images to do so. Otherwise, only historical processed images will be used.
+- `?showgrid`: **the exception; we will want to use this on one of the production screens** shows the grid / historical view rather than the json / latest view.
```

Comment:
I see how hinting at the query string question mark prefix could be helpful but also seems somewhat misleading as this is a list of parameters names available. Can more than one param be applied at a time? In that case, the url would be, for example, `http://localhost:8080?showgrid&controls&timing=fast`, not `http://localhost:8080?showgrid?controls?timing=fast`.

Maybe approach would be to indicate a query string can be added to the base url with multiple (assuming that is true) params from the list below and then provide an example url for reference?

Reply:
@paulfarning Thanks for the review! I just fixed the punctuation issues and updated the query string section to try to make it more clear - let me know what you think.

Reply:
Oh duh - just looked at your comment again. Yes, I was incorrectly adding these with `?` between them, but I see now that the and `&` symbol is the correct was to pass multiple parameters to a query string - I will update. 👍 `http://localhost:8080/?showgrid&prepopulate&zoom`